### ISSUE TITLE
FontPicker

### ISSUE THREAD

**Issue by unmick** [getgrav/grav-plugin-admin#1333]
Hi, 

More than an issue, it's an enhancement request :-)

As the colorpicker, i would like a "fontpicker" .
Does someone have already done or try something ? 

I have a look at the Gantry theme where it exists but i have to overwrite the admin plugin and i am not sure to do it successfully.

Thanks

**Comment by diomed** [comment]
yeah, it would be totally lovely if we could just write our desired font name and grav would get it from Google Fonts

**Comment by rhukster** [comment]
We actually have an amazing font picker (and icon picker) in Gantry framework. It's long been the plan to port them to Grav admin, but they are quite complex and we've just not had the time yet.

**Comment by paulmassen** [comment]
A year ago, I I forked the assets plugin for selecting assets from cdnjs. I never finished it, but if it has some interest, it would definitely be possible to add the font selection feature.
Here is how it worked:
![cssjspicker](https://user-images.githubusercontent.com/12809352/36308958-03a6a7de-1323-11e8-878c-6c572e61896b.gif)
